app/views/govuk_publishing_components/components/docs/share_links.yml

```diff
@@ -65,6 +65,16 @@ examples:
           icon: 'facebook'
         }
       ]
+  with_custom_visually_hidden_text:
+    data:
```

Comment:
If I'm reading the code right, it looks like if no visually hidden text is provided it defaults to 'Share on'. I think that's worth documenting here with a description, otherwise we might end up with share links that duplicate it unknowingly e.g. '(Share on) Share on Facebook'.

Reply:
description added